lib/pages/main/main_vc.dart

```diff
@@ -19,53 +19,38 @@ class _MainVCState extends State<MainVC> {
     SettingsPage(),
   ];
 
-  void _onItemTapped(int index) {
-    setState(() {
-      _selectedIndex = index;
-    });
-  }
-
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: _pages[_selectedIndex],
-      bottomNavigationBar: ClipRRect(
-        borderRadius: BorderRadius.only(
-          topLeft: Radius.circular(40.0),
-          topRight: Radius.circular(40.0),
-        ),
-        child: BottomAppBar(
-          color: Colors.grey[350],
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceAround,
-            children: <Widget>[
-              _buildIconButton(Icons.home, 0),
-              _buildIconButton(Icons.map, 1),
-              _buildIconButton(Icons.file_copy, 2),
-              _buildIconButton(Icons.person, 3),
-            ],
+      bottomNavigationBar: NavigationBar(
+        onDestinationSelected: (int index) {
+          setState(() {
+            _selectedIndex = index;
+          });
+        },
+        indicatorColor: Colors.amber,
```

Comment:
I forgot to mention that this should be the colorprimary (see themedata) with opacity of 0.24